app/src/main/java/com/infomaniak/mail/data/models/correspondent/Recipient.kt

```diff
@@ -17,7 +17,9 @@
  */
 package com.infomaniak.mail.data.models.correspondent
 
+import android.os.Parcel
 import io.realm.kotlin.types.EmbeddedRealmObject
+import kotlinx.parcelize.Parceler
```

Comment:
Remove `@Suppress("PROPERTY_WONT_BE_SERIALIZED")` now that there is a manual Parceler